src/__Private/LintRunErrorHandler.php

```diff
@@ -0,0 +1,32 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST\__Private;
+
+use type Facebook\CLILib\OutputInterface;
+use namespace Facebook\HHAST\Linters;
+
+/**
+ * Interface for creating custom handlers for lint errors that are found during
+ * a hhast-lint CLI run. Some examples of hander actions are generating human
```

Comment:
Missed this in the last PR, but I don't think this file only applies to CLI runs?

Reply:
yep